samples/hello-spring-boot/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+group = "com.spring.boot"
+version = "1.0.0-SNAPSHOT"
+
+plugins {
+    val kotlinVersion = "1.2.21"
+    id("org.springframework.boot") version "2.0.0.RELEASE"
+    id("org.jetbrains.kotlin.jvm") version kotlinVersion
+    id("org.jetbrains.kotlin.plugin.spring") version kotlinVersion
```

Comment:
Could be replaced with `kotlin("plugin.spring") version kotlinVersion`
(same for `jvm` of course)